Dockerfile

```diff
@@ -12,13 +12,11 @@ FROM python:${PYTHON_VERSION} as run
 WORKDIR /app
 
 ENV PYTHONUNBUFFERED=1
+ENV ENV_ENGINE="default"
```

Comment:
The environment variable `ENV_ENGINE` is set to 'default'. Ensure that this default value is appropriate for your application, or consider overriding it with a more specific value during deployment.